src/resolvers/category.resolver.ts

```diff
@@ -1,13 +1,26 @@
-import { Resolver, Query } from '@nestjs/graphql';
-import { CategoryType } from '../types/category';
+import { Resolver, Query, Args } from '@nestjs/graphql';
+import { CategoryType, CategoryInputType } from '../types/category';
 import CategoryService from '../services/category.service';
 
 @Resolver(_of => CategoryType)
 export default class CategoryResolver {
   constructor(private readonly categoryService: CategoryService) {}
 
-  @Query(_returns => [CategoryType])
-  async test() {
-    return await this.categoryService.getCategory();
+  @Query(_returns => [CategoryType], {
+    name: 'category',
+    description: 'category 정보를 가져옵니다.',
+    nullable: true,
+  })
+  async getCategory(
```

Comment:
Category는 배열이니깐 복수형이 더 낫지않을까요
```suggestion
  async getCategories (
```

Reply:
수정하였습니다!